frontend/src/App.js

```diff
@@ -1,16 +1,20 @@
 import React, { useState, useEffect } from 'react';
-import '@trussworks/react-uswds/lib/uswds.css';
 import '@trussworks/react-uswds/lib/index.css';
+import 'uswds/dist/css/uswds.css';
```

Comment:
Why this uswds and not from trussworks?

edit: because that component isn't yet in trussworks? Did trussworks include anything in their styling that we now need to account for?

Reply:
From https://github.com/trussworks/react-uswds/blob/main/docs/styles_and_assets.md trussworks puts all their custom CSS in the `@trussworks/react-uswds/lib/index.css` file and offers up a version of the `uswds` CSS file in `@trussworks/react-uswds/lib/uswds.css`. The CSS for the stepper isn't in the `uswds` CSS file imported from trussworks (I'm sure it will end up there eventually), so we have to import CSS directly from the `uswds` package to pull in those styles

Reply:
Ah, perfect. Should the base css be imported before the truss customizations? I always forget if that's important or not, or if it's impacted by getting pulled in and compiled by webpack.

Reply:
It could be important, I'll update the import order to match what trussworks has in their docs to be sure